spec/controllers/pages_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe PagesController, type: :controller do
+
```

Comment:
`Rubocop` is complaining because `Extra empty line detected at block body beginning.`, so `delete` that line and `rubocop` should be happy!